packages/ui/v2/core/layouts/SettingsLayout.tsx

```diff
@@ -89,18 +93,160 @@ const useTabs = () => {
 };
 
 const SettingsSidebarContainer = ({ className = "" }) => {
+  const { t } = useLocale();
   const tabsWithPermissions = useTabs();
+  const [teamMenuState, setTeamMenuState] =
+    useState<{ teamId: number | undefined; teamMenuOpen: boolean }[]>();
+
+  const { data: teams } = trpc.useQuery(["viewer.teams.list"]);
+
+  useEffect(() => {
+    if (teams) {
+      const teamStates = teams?.map((team) => ({ teamId: team.id, teamMenuOpen: false }));
+      setTeamMenuState(teamStates);
+    }
+  }, [teams]);
```

Comment:
Use this state to determine if the team specific collapsable menu should be open or closed